deploy/vm/test/ha_pair_failover_test/roles/check-hdb-status/tasks/main.yml

```
@@ -0,0 +1,13 @@
+- pause:
+     minutes: 2
+- name: Run the landscapeconfig script to get the configuration table 
+  shell: . ~/.bashrc && python /usr/sap/"{{ sap_sid }}"/HDB"{{ instance_num }}"/exe/python_support/landscapeHostConfiguration.py > /tmp/configOut.txt
+  ignore_errors: True
```

Why are we ignoring errors? It seems like a test should not ignore errors

---

already explained above